py/build.gradle

```diff
@@ -3,6 +3,13 @@ apply from: "$rootDir/gradle/utils.gradle"
 ext {
     FS = File.separator
     FPS = File.pathSeparator
+
+    def command = ["python", "py/pythonVersion.py"]
```

Comment:
yes that's what I meant, tried it and didn't work but maybe I passed the parameters incorrectly (i.e. "-c" and "(STUFF)" separately and it should be together as one string, maybe applied too many quotes etc) hm... Though I think I tried quite a fee combinations